Widgets/WidgetBase/VarmintWidgetChildren.cs

```diff
@@ -86,13 +86,7 @@ public virtual void AddChild(VarmintWidget widget, bool suppressChildUpdate = fa
         //--------------------------------------------------------------------------------------
         public virtual void InsertChild(VarmintWidget widget, bool suppressChildUpdate = false)
         {
-            List<VarmintWidget> updatedChildren = new List<VarmintWidget>();
-            updatedChildren.Add(widget);
-            widget.Parent = this;
```

Comment:
Keep this line, it is still needed. 
Think of it this way:  The InsertChild and AddChild methods need to behave in EXACTLY the same way except that:  Insert puts the child at the front of the list, and add puts the child at the end of the list.